launcher/java/JavaUtils.cpp

```diff
@@ -237,22 +231,25 @@ QList<JavaInstallPtr> JavaUtils::FindJavaFromRegistryKey(DWORD keyType, QString
                                       KEY_READ | KEY_WOW64_64KEY, &newKey) == ERROR_SUCCESS)
                     {
                         // Read the JavaHome value to find where Java is installed.
-                        value = new char[0];
+                        delete [] value;
+                        value = new TCHAR[0];
                         valueSz = 0;
```

Comment:
This is where valueSz should actually be declared.
```suggestion
                        DWORD valueSz = 0;
```